github-runner-manager/src/github_runner_manager/configuration/base.py

```diff
@@ -131,12 +145,18 @@ class SSHDebugConnection(BaseModel):
         port: The SSH relay server port.
         rsa_fingerprint: The host SSH server public RSA key fingerprint.
         ed25519_fingerprint: The host SSH server public ed25519 key fingerprint.
+        use_runner_http_proxy: Whether to use runner proxy for the SSH connection.
+        local_http_proxy_host: Local host to use for proxying.
+        local_http_proxy_port: Local port to use for proxying.
```

Comment:
```suggestion
        local_proxy_host: Local host to use for proxying.
        local_proxy_port: Local port to use for proxying.
```

iiuc its a general tcp proxy

Reply:
not sure, the proxy is http, but we use it to just tunnel a tcp connection....

Reply:
but isn't 

```
socat -v TCP-LISTEN:{{ssh_debug_info.local_http_proxy_port}},fork,bind={{ssh_debug_info.local_http_proxy_host}} PROXY:{{runner_proxy_config.proxy_host}}:{{ssh_debug_info['host']}}:{{ssh_debug_info['port']}}{% if runner_proxy_config.proxy_port %},proxyport={{runner_proxy_config.proxy_port}}{% endif %}
```

creating a tunnel which listens on local_http_proxy_port , which is in theory independent of the http protocl? Shouldn't  the name not be rather `local_socat_tunnel_port` ?

At least for me the line `TMATE_SERVER_HOST={{ssh_debug_info.local_http_proxy_host}}` in env.j2 reads a bit confusing, as I would not expect the TMATE_SERVER_HOST to be a http proxy.

Reply:
Updated to remove the http prefix

Reply:
I think you removed proxy instead of http

Reply:
ouch